scripts/execute_ebpf_cicd_stress_tests.ps1

```diff
@@ -0,0 +1,36 @@
+﻿# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+
+param ([parameter(Mandatory=$false)][string] $AdminTarget = "TEST_VM",
```

Comment:
How different is this file from `execute_ebpf_cicd_tests.ps1`? If it is very similar to the existing file, is it possible to update `execute_ebpf_cicd_tests.ps1` and control which test to run via a parameter passed to the script?